cloud: update export data from tidb cloud doc (#12700)
pingcap/docs: tidb-cloud/export-data-from-tidb-cloud.md

    <div label="Serverless Tier">

    ```shell
    tiup dumpling -h ${tidb_endpoint} -P 4000 -u ${user} -p ${password} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql --consistency none
    ```
   
   > **Note:**
   > 

>    > For exporting serverless tier data, you must ensure that the dumpling version is at least v6.5.0.

Reviewer: ```suggestion
   > To export Serverless Tier cluster data, you must ensure that the dumpling version is at least v6.5.0.
```

>    > To export Serverless Tier cluster data, you must ensure that the dumpling version is at least v6.5.0.

Reviewer 2: ```suggestion
   > To export Serverless Tier cluster data, you must ensure that the Dumpling version is at least v6.5.0.
```

   >

>    > When using dumpling:v6.5.0, you need to set `--ca=${ca_path}` in the command. To find the CA root path on your system, see [TLS Connections to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md#root-certificate-default-path).

Reviewer: What about other Dumpling versions? For example v6.6.0, do users need to set `--ca=${ca_path}` in the command?

Author: @qiancai No, they needn't. Only v6.5.0 needs to set the CA path.

>    > When using dumpling:v6.5.0, you need to set `--ca=${ca_path}` in the command. To find the CA root path on your system, see [TLS Connections to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md#root-certificate-default-path).

Reviewer 2: ```suggestion
   > If your Dumpling version is v6.5.0, you need to set `--ca=${ca_path}` in the command. To find the CA root path on your system, see [TLS Connections to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md#root-certificate-default-path).
```

>    > To export Serverless Tier cluster data, you must ensure that the Dumpling version is at least v6.5.0.
>    >
>    > If your Dumpling version is v6.5.0, you need to set `--ca=${ca_path}` in the command. To find the CA root path on your system, see [TLS Connections to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md#root-certificate-default-path).

Reviewer: ```suggestion
   > To export Serverless Tier cluster data, you must ensure that the Dumpling version is at least v6.5.0. If your Dumpling version is v6.5.0, you also need to set `--ca=${ca_path}` in the command. To find the CA root path on your system, see [TLS Connections to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md#root-certificate-default-path).
```

    - `-u`: The TiDB cluster user.
    - `-p`: The TiDB cluster password.
    - `-F`: The maximum size of a single file.
    - `-o`: The export directory.
    - `--filetype`: The exported file type. The default value is `sql`. You can choose from `sql` and `csv`.

>     - `--consistency`: The data consistency. The default value is `auto`. For Serverless Tier, you must set it to `none`.

Author: @zhangyangyu No. `lock` needs `RESTRICTED_VARIABLES_ADMIN` privilege.
Here is the error output:
```
dump failed: sql: SELECT @@tidb_config;: Error 1227 (42000): Access denied; you need (at least one of) the RESTRICTED_VARIABLES_ADMIN privilege(s) for this operation
```
